run.bash

```diff
@@ -63,6 +63,25 @@ if ! install_lava "${LAVA_VERSION}"; then
 	exit 1
 fi
 
+# Install yq.
+check_status (){
+	local status=$1
+	local error_msg=$2
+	if [[ $status -ne 0 ]]; then
+		echo $error_msg
+		echo "status=${status}" >> "${GITHUB_OUTPUT}"
+		exit $status
+	fi
+}
+
+metrics=$(mktemp)
+
+sudo wget -q https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64 -O /usr/local/bin/yq
+check_status $? "error: could not install yq"
+sudo chmod +x /usr/local/bin/yq
```

Comment:
Given that we are not using `check_status` anywhere else, I would just remove it. I would also pin the the `yq` version and download the .tar.gz which is significantly smaller.

```suggestion
# Install yq.

if (curl -LsSf "https://github.com/mikefarah/yq/releases/download/${YQ_VERSION}/yq_linux_amd64.tar.gz" | sudo tar -xz -C "/usr/local/bin" lava && chmod +x /usr/local/bin/yq) 2> /dev/null; then
	echo 'error: could not install yq' >&2
	exit 1
fi

metrics=$(mktemp)
```

and I'd add something like this at the beginning of the script.

```bash
YQ_VERSION=${YQ_VERSION:-v4.44.1}
```

Reply:
done in 570a1fbff3cf35d974790c7dfea00872905d5c55